src/content/tools/pub/cmd/pub-token.md

```diff
@@ -42,33 +44,45 @@ Enter secret token: <Type token on stdin>
 ```
 
 :::note
-The token is input on stdin rather than as a command line option to avoid it
-ending up in the shell history such as `~/.bash_history`.
+The token takes input on `stdin` rather than as a command line option
+to keep the token out of the shell history.
 :::
 
-In a scripting situation you can store the secret in an environment variable and
-use `dart pub token add <hosted-url> --env-var <ENV_VAR_NAME>`.
+### Add a credential for all sessions
 
-```console
-$ dart pub token add https://other-package-repo.com/ --env-var TOKEN_VAR
-Requests to "https://other-package-repo.com/" will now be authenticated using the secret token stored in the environment variable "TOKEN_VAR".
-```
+In a script, you can store the secret in an environment variable.
+
+If you choose to store your secret in an environment variable,
+find a way to hide the secret from your shell history.
+To explore one way of doing this, consult [this post on Medium][zsh-post].
+If you add a local environment variable, you need to restart any open
+consoles to enable that new variable.
+
+Most CI environments can inject secrets into an environment variable.
+To learn how, consult documentation for [GitHub Actions][]] or
+[GitLab][] as examples.
```

Comment:
```suggestion
Most CI environments can inject tokens into an environment variable.
To learn how, consult documentation for [GitHub Actions][] or
[GitLab][] as examples.
```

Reply:
**comment:** Agreed, but CI systems usually call them secrets. I know, it's just synonym city here.

Reply:
You missed the extra bracket after GitHub Actions